content/Assets/Styles/base/_anchor.scss

```diff
@@ -25,17 +26,22 @@ a {
         @include rem(padding-left, $spacing-default);
         @include rem(padding-right, $spacing-default);
         @include rem(padding-top, $spacing-small);
-        
-        @include rem(font-size, $font-size-h3);
+
+        @include themify($themes) {
+            font-size: themed('fontSize.h3');
```

Comment:
Does this need to be REMified as well as themed, or can we rely on the setting always being set in REM? Genuine question as I'm not sure :)

Reply:
If I recall correctly the `rem` mixin also adds a `px` fallback value, even when a `rem` value is provided.

Reply:
I read @Newnab's question backwards (thought it _was_ using `@include rem`)!

@peterkeating I believe I know the issue you're referring to as having a themed value passed into a mixin is something that has tripped me up a few times, I'll be interested to know your findings here.

Reply:
Looks like I must have had a typo or something because it works as shown below.

```
@include themify($themes) {
    @include rem(font-size, themed('fontSize.small'));
}
```